lib/bookmark_web/controllers/archive_controller.ex

```diff
@@ -136,7 +129,7 @@ defmodule BookmarkWeb.ArchiveController do
       [_err, id] = String.split(List.first(regex_result), "archive/")
       user = conn.assigns.current_user
 
-      Bookmark.Archives.create_archive(%{name: id, comment: ""}, user)
+      Bookmark.Archives.create_archive(%{name: id, comment: "", title: url}, user)
```

Comment:
It's unclear in this context what "title: url" refers to

Reply:
I see what this is now. We should not use a url as a title though. Why do we need to provide a title before archive is created? We could pull the actual page title from the index.json we get back from archivebox. So this title would match the TITLE tag of the target archive.

Reply:
Done!